### ISSUE TITLE
revoke button not visible

### ISSUE THREAD

**Issue by pulsarinformatique** [Kicktemp/kickgdpr#5]
Hi
Thanks for this great plugin but I can't see any revoke tab ("cookies policy") once you accepted or refused the cookies with CHROME. With Firefox I can see it.

thanks

cyril

**Comment by tecpromotion** [comment]
Hi @pulsarinformatique 
Can you please give me a link where it is not displayed in Chrome. Thank you.

**Comment by pulsarinformatique** [comment]
Hi 
yes : https://www.eden-love.com/
thanks

**Comment by tecpromotion** [comment]
I use Chrome (latest version) and the button is shown bottom left.

![screenshot 2018-09-18 09 36 30](https://user-images.githubusercontent.com/1752513/45671720-7b80f380-bb26-11e8-81ba-52dd1930cb9f.png)
![screenshot 2018-09-18 09 36 38](https://user-images.githubusercontent.com/1752513/45671721-7c198a00-bb26-11e8-9211-85fdfde76989.png)

**Comment by pulsarinformatique** [comment]
yes you can see it but only once. if you F5 it no longer appears

**Comment by nielsnuebel** [comment]
@pulsarinformatique https://cookieconsent.insites.com/documentation/javascript-api/#revoke_options

![image](https://user-images.githubusercontent.com/3214077/45676483-1df2a400-bb32-11e8-9748-87e0c46f0c22.png)
Give us a few hours to test.

**Comment by tecpromotion** [comment]
Dev environment only.
